### DISCUSSION TITLE
Does ESP32-C3-WROOM work with this library?

### DISCUSSION THREAD

**Discussion by clarkbremer** [pschatzmann/ESP32-A2DP#221] [Q&A] [answered]
The ESP32 development board I've been using has an ESP32-WROOM-32D module on it.  I'm thinking about placing the module directly onto my own PCB (without the development board), but that module is flagged as "not recommended for new designs".  So I'm looking at using an ESP32-C3-WROOM-02 module (https://www.espressif.com/sites/default/files/documentation/esp32-c3-wroom-02_datasheet_en.pdf).  Any reason why the A2DP library wouldn't work on that?  I notice it's a single core.  But they're only $2!  Anything else I should be thinking about?

**Comment by pschatzmann** [comment, marked as answer]
The C3 does not support the Standard Bluetooth stack. 
It only allows BLE. So A2DP will not be possible

**Reply by clarkbremer** [reply]
Thanks.  I thought that sounded too good to be true.  The 32D is about $4, and I've found dev boards with that module for $6.  Not sure I'll get a cost savings that way.
